containers/proxy-helm/values.yaml

```diff
@@ -1,6 +1,15 @@
 
-repository: registry.opensuse.org/uyuni
-version: latest
+defaultRegistry: registry.opensuse.org/uyuni
+defaultVersion: latest
+
+## Allows to override the default URI for an image if defined
+## Requires a full URI in a form of <registry>/<image>/<version>
+## Available service identifiers are:
+##   proxy-httpd, proxy-salt-broker, proxy-squid, proxy-ssh, proxy-tftpd
+##
+images:
+    # <service_identifier>: <customImageURI>
+
```

Comment:
I would prefer to list in comments all values like it is in environment file for systemd.

I did not check helm behavior when include file is missing, but could we do something I proposed below with systemd to include files which by default does not exists?